src/rpc/blockchain.cpp

```diff
@@ -363,15 +371,17 @@ UniValue getblockheader(const UniValue& params, bool fHelp)
 
 UniValue getblock(const UniValue& params, bool fHelp)
 {
-    if (fHelp || params.size() < 1 || params.size() > 2)
+    if (fHelp || params.size() < 1 || params.size() > 3)
         throw runtime_error(
-            "getblock \"hash\" ( verbose )\n"
+            "getblock \"hash\" ( verbose ) ( listtransactions )\n"
             "\nIf verbose is false, returns a string that is serialized, hex-encoded data for block 'hash'.\n"
             "If verbose is true, returns an Object with information about block <hash>.\n"
+            "If listtransactions is true, a list of the IDs of all the transactions included in the block will be shown.\n"
             "\nArguments:\n"
             "1. \"hash\"          (string, required) The block hash or height\n"
             "2. verbose           (boolean, optional, default=true) true for a json object, false for the hex encoded data\n"
-            "\nResult (for verbose = true):\n"
+            "3. listtransactions  (boolean, optional, default=true) true to get a list of all txns, false to get just txns count\n"
+            "\nResult (for verbose = true, listtransactions = true):\n"
```

Comment:
nit:  perhaps change this to say "...  false to get a count only"

Reply:
think your wording is better, going to change.